### DISCUSSION TITLE
Custom event fields in dashboard and hunt interface

### DISCUSSION THREAD

**Discussion by rahatUlAin** [Security-Onion-Solutions/securityonion#12368] [Unsupported Versions] [answered]
I need to view relevant fields as columns in hunt and dashboard interface. Earlier I added event 
fields for my logs in /opt/so/saltstack/local/salt/soc/files/soc/hunt.eventfields.json like this https://github.com/Security-Onion-Solutions/securityonion/blob/dev/salt/soc/files/soc/hunt.eventfields.json

Is there a way in security onion 2.4 to still be able to define the columns to show.

**Comment by reyesj2** [comment, marked as answer]
https://docs.securityonion.net/en/2.4/dashboards.html#table You can setup your dashboards to use custom columns using the ``` | table xxx yyy zzz``` format.

![image](https://github.com/Security-Onion-Solutions/securityonion/assets/94730068/90d6277d-9b0f-4e95-bdc1-728a2ed8f03a)
